README.md

```diff
@@ -60,20 +57,13 @@ feature. Use the ``--help`` option for more information.
 ### Audit logs
 Sadly, the org audit log does not have an API, so we'll screen scrape a little.
 
-#### audit-log-export.js
-So here's a little bit of JavaScript you can run on any audit log page to export the data into a somewhat more digestible JSON format.
-
-Example URL: https://github.com/orgs/acmecorp/audit-log
-
-Yes, you have to go to the next page and re-run this to get another file.
-
 #### hooks.py
 Analyzes a list of audit log export files (from the JS script) for hook/service creation/deletion and provides a summary. Use it to show commonly used apps/services/webhooks across the org.
 
 ### old_repos.py
 Generate a list of empty (should be deleted) repositories as well as untouched repos (might need to be archived).
 
 ## License
-This code is free software and licensed under an MPL-2.0 license. &copy; 2015-2018 Fred Wenzel and others. For more information read the file ``LICENSE``.
```

Comment:
This is flattering, though I think by now you deserve waaaaay more credit, @hwine !